include/mbedtls/check_config.h

```diff
@@ -962,6 +962,11 @@
 #error "MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH defined, but not all prerequisites"
 #endif
 
+#if defined(MBEDTLS_SSL_SESSION_TICKETS) && \
+    !( defined(MBEDTLS_GCM_C) || defined(MBEDTLS_CCM_C) || defined(MBEDTLS_CHACHAPOLY_C) || defined(MBEDTLS_NIST_KW_C) )
```

Comment:
I'm sorry, but I really think `NIST_KW` shouldn't be part of the list, see my [previous comment](https://github.com/Mbed-TLS/mbedtls/pull/6374/files/8d4b24102883315d32135af4e30f5857ee14c582..ce5b68c7a304ad5c9b15613925ae47f0b923ae15#diff-4fc3811c7c53e1746e1056ac38a6462ad2160077a59898bbd2bc31f5938ac185).